plenum/test/cli/test_genesis_txns.py

```diff
@@ -1,56 +1,102 @@
+        '{{"node_ip": "localhost", "node_port": "9701", "client_ip": "localhost", '
         '"client_port": "9702", '
-        '"alias": "PhilNode"}')
+        '"alias": "PhilNode"}}'.format(node=PlenumTransactions.NODE.name))
     assert len(cli.genesisTransactions) == oldGenTxns + 1
     assert cli.lastCmdOutput == "Genesis transaction added"
     cli.enterCmd(
-        'add genesis transaction NODE for 59d9225473451efffe6b36dbcaefdbf7b1895de62084509a7f5b58bf01d06419 by 59d9225473451efffe6b36dbcaefdbf7b1895de62084509a7f5b58bf01d06418 with data '
-        '{"node_ip": "localhost", "node_port": "9701", "client_ip": "localhost", '
+        'add genesis transaction {node} for 59d9225473451efffe6b36dbcaefdbf7b1895de62084509a7f5b58bf01d06419 by 59d9225473451efffe6b36dbcaefdbf7b1895de62084509a7f5b58bf01d06418 with data '
+        '{{"node_ip": "localhost", "node_port": "9701", "client_ip": "localhost", '
         '"client_port": "9702", '
-        '"alias": "PhilNode"}')
+        '"alias": "PhilNode"}}'.format(node=PlenumTransactions.NODE.name))
     assert len(cli.genesisTransactions) == oldGenTxns + 2
     assert cli.lastCmdOutput == "Genesis transaction added"
     return cli
 
 
-def testNewStewardGenTxn(newStewardsAdded):
+@pytest.fixture("module")
+def newNodesAddedByValue(cli):
+    oldGenTxns = len(cli.genesisTransactions)
+    cli.enterCmd(
+        'add genesis transaction {node} for 59d9225473451efffe6b36dbcaefdbf7b1895de62084509a7f5b58bf01d06420 by 59d9225473451efffe6b36dbcaefdbf7b1895de62084509a7f5b58bf01d06420 with data '
+        '{{"node_ip": "localhost", "node_port": "9701", "client_ip": "localhost", '
+        '"client_port": "9702", '
+        '"alias": "PhilNode"}}'.format(node=NODE))
+    assert len(cli.genesisTransactions) == oldGenTxns + 1
+    assert cli.lastCmdOutput == "Genesis transaction added"
+    cli.enterCmd(
+        'add genesis transaction {node} for 59d9225473451efffe6b36dbcaefdbf7b1895de62084509a7f5b58bf01d06421 by 59d9225473451efffe6b36dbcaefdbf7b1895de62084509a7f5b58bf01d06420 with data '
+        '{{"node_ip": "localhost", "node_port": "9701", "client_ip": "localhost", '
+        '"client_port": "9702", '
+        '"alias": "PhilNode"}}'.format(node=NODE))
+    assert len(cli.genesisTransactions) == oldGenTxns + 2
+    assert cli.lastCmdOutput == "Genesis transaction added"
+    return cli
+
+
+def testNewStewardGenTxnByName(newStewardsAddedByName):
     pass
 
 
-def testNewNodeGenTxn(newNodesAdded):
+def testNewStewardGenTxnByValue(newStewardsAddedByValue):
     pass
 
 
-def testCreatePoolTxnFle(newStewardsAdded, newNodesAdded):
-    cli = newNodesAdded
-    assert len(cli.genesisTransactions) == 4
+def testNewNodeGenTxnByName(newNodesAddedByName):
+    pass
+
+
+def testNewNodeGenTxnByValue(newNodesAddedByValue):
+    pass
```

Comment:
Fixed.
newNodesAddedByValue is a fixture which had some asserts in it (that's how most of the tests is written).
I re-factored it to have asserts in the tests instead of the fixture (I think all tests need to have explicit asserts within the test function, not inside a fixture).